### DISCUSSION TITLE
테스트 접속 링크로 서로 만나게 할 수 없는건가요?

### DISCUSSION THREAD

**Discussion by devinhyeok** [naverz/zepeto-studio-kor#274] [5. Q&A]
![image](https://user-images.githubusercontent.com/94621970/151120939-8d5eaad5-db89-4914-be2f-0ecc3b4c73e4.png)
테스트 접속 링크로 두기기에 같은 QR로 접속하게 했더니
따로 방을 파고 초대도 할 수 없습니다.
그럼 멀티 환경에서의 테스트는 어떻게 해야하나요?

**Comment by x4cf** [comment]
테스트 접속 링크와 QR은 다른 기능입니다.
QR의 경우는 멀티테스트가 정상적으로 작동됩니다.
테스트 접속 링크도 정상적으로 작동됩니다.

QR테스트라면 같은 네트워크에서 테스트하는게 맞을까요?

**Reply by devinhyeok** [reply]
https://github.com/naverz/zepeto-studio-kor/discussions/248
테스트 접속 링크를 QR로 바꿔서 쓰라고 하길래 이렇게 해서 접속에 성공했습니다.
다만 멀티플레이 확인을 위해 같은 방에 초대를 하려고 했으나 방법을 찾을 수 없습니다.

**Comment by moondory77-naverz** [comment]
안녕하세요. 
현재 Pretest 상태의 월드패키지에 대해, 초대기능을 지원하지 않고 있습니다.
다음 제페토 앱 업데이트(3.9.7) 일정에 맞춰, 해당 기능을 지원드릴 예정이니 참고 부탁드리겠습니다.
감사합니다.
